packages/api-client/src/books.ts

```diff
@@ -0,0 +1,15 @@
+import { GetBookProgressResponseBody } from '@translation/api-types';
+import ApiClient from './client';
+
+export default class Books {
+  constructor(private readonly client: ApiClient) {}
+
+  findProgress(
```

Comment:
We can make this a method in the languages module and name it `findBookProgress`